tests/test_data/endpoints.py

```diff
@@ -0,0 +1,5 @@
+favorites_url = 'https://my.prom.ua/cabinet/user/favorites'
+velosipednye_shiny_url = 'https://prom.ua/Velosipednye-shiny'
+
+gorest_users_url = 'https://gorest.co.in/public/v2/users'
+access_token = 'access-token=4aa39ba4771752ba989306eb2ef68943963538efeaf4d2feedb7b22ebe3368a7'
```

Comment:
the same issue with sensitive data as mentioned above.
The github history keep all your changes, so once your password appeared in the code everybody can retrieve it and stole your accounts.

Reply:
It doesn't matter. I don't value this password much )